src/Parser/Priest/Discipline/CHANGELOG.js

```diff
@@ -1,4 +1,5 @@
 export default `
+27-10-2017 - Refactored disc module. (Gao)
```

Comment:
Not sure if internal changes that don't have external changes should be in this changelog.